src/data-source.js

```diff
@@ -28,13 +28,15 @@ function dataSource(params) {
   }
 
   // expect the _ow_path to start with /https:// or /https%3a%2f%2f
-  if (path.startsWith('/https%3A%2F%2F')) {
+  if (path.startsWith('/https%3A%2F%')) {
```

Comment:
I think, when the URL is escaped, we don't want the hacky behaviour. 
besides, it would fail anyways, if there is only one slash.